osu.Game/Screens/Ranking/UserTagControl.cs

```diff
@@ -42,14 +42,20 @@ public partial class UserTagControl : CompositeDrawable
 
         private readonly Bindable<APIBeatmap?> apiBeatmap = new Bindable<APIBeatmap?>();
 
-        private AddNewTagUserTag addNewTagUserTag = null!;
+        private AddNewTagUserTag? addNewTagUserTag;
+
+        /// <summary>
+        /// Determines whether the user can modify the contained tags
+        /// </summary>
+        private readonly bool writable;
 
         [Resolved]
         private IAPIProvider api { get; set; } = null!;
 
-        public UserTagControl(BeatmapInfo beatmapInfo)
+        public UserTagControl(BeatmapInfo beatmapInfo, bool writable)
```

Comment:
would strongly prefer that this be an `init` property rather than a ctor parameter